### PR TITLE
bring back delete

### PR DESCRIPTION
**Pull request by SamSamskies**
resolves https://github.com/coracle-social/coracle/issues/264

### CODE REVIEW

File: src/engine/commands.ts [hunk 1]
```diff
@@ -647,8 +647,12 @@ export const publishCommunitiesList = addresses =>
...
 // Deletes
 
-export const publishDeletion = ({kind, address = null, id = null}) => {
-  const tags = [["k", String(kind)]]
+export const publishDeletion = ({kind = null, address = null, id = null}) => {
+  const tags = []
+
+  if (kind) {
+    tags.push(["k", String(kind)])
+  }
```
Comment (SamSamskies) [inline src/engine/commands.ts:655]: Do you prefer to always include the kind? I didn't see this in the spec so was wondering what's the use case for including it.

File: src/app/views/NoteDelete.svelte [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+  import Anchor from "src/partials/Anchor.svelte"
+  import Content from "src/partials/Content.svelte"
+  import {router} from "src/app/util/router"
+  import {deleteEventById} from "src/engine"
+
+  const onCancel = () => router.pop()
+
+  const onConfirm = () => {
+    deleteEventById($$props.eid)
```
Comment (staab) [inline src/app/views/NoteDelete.svelte:10]: I prefer `export let eid` when possible

File: src/app/views/NoteDelete.svelte [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    router.pop()
+  }
+</script>
+
+<Content size="lg">
```
Comment (staab) [inline src/app/views/NoteDelete.svelte:15]: Content is deprecated, use `FlexColumn` instead

File: src/engine/commands.ts (hunk 1 above)
Comment (staab) [reply src/engine/commands.ts:655]: Yeah, there's an open PR: https://github.com/nostr-protocol/nips/pull/1141/files it's just so deletes can be requested more efficiently.

**Review by staab — CHANGES_REQUESTED** [review]
Looks good aside from some minor things, I'm on vacation so it'll probably sit in limbo for a couple weeks before I merge and get back to you with the next thing.

**Pull request merged**